src/user/UserInterface.java

```diff
@@ -1,19 +1,46 @@
 package user;
 
+import flight.Flights;
+import airport.Airports;
 import plans.FlightPlan;
 import plans.FlightPlans;
 import plans.Reservation;
 import plans.SearchParams;
+import utils.Date;
+import utils.Time;
 
 public class UserInterface {
 	public void DisplaySearch(){
-		
+		System.out.println("The valid airport codes are as follows: ");
+		for(int i = 0; i< airport.Airports.instance.size(); i++){
+			System.out.print(airport.Airports.instance.get(i).getCode());
+			if((i+1) == airport.Airports.instance.size()){
+				System.out.print(".\n");
+			}
+			else if((i+1) % 5 == 0 && i != 0){
```

Comment:
if i+1 is divisible by 5 then i is definitely not zero, so the condition term `i != 0` is redundant. 😄